Dockerfile

```diff
@@ -1,14 +1,21 @@
-FROM registry.access.redhat.com/ubi8/python-39
+FROM quay.io/centos/centos:stream9-development
 
 ARG USER_ID=${USER_ID:-1001}
+ARG APP_DIR=${APP_DIR:-/app}
 ARG DEVEL_COLLECTION_LIBRARY=0
-WORKDIR $HOME
 
 USER 0
-RUN dnf install -y java-17-openjdk-devel
+RUN useradd -u $USER_ID -d $APP_DIR appuser
+WORKDIR $APP_DIR
+COPY . $WORKDIR
+RUN chown -R $USER_ID $APP_DIR
+RUN dnf install -y java-17-openjdk-devel python3-pip
+
+USER $USER_ID
 ENV JAVA_HOME=/usr/lib/jvm/java-17-openjdk
+ENV PATH="${PATH}:$APP_DIR/.local/bin"
 RUN pip install -U pip \
-    && pip install ansible \
+    && pip install ansible-core \
```

Comment:
IMO I prefer to use `ansible` package to have a full ansible experience.

Reply:
I agree the `ansible` package is usually preferable, but in this case it adds an extra ~500mb due to all of the collections that get installed alongside it.